### PR TITLE
Fix duplicated role settings when several roles exists

### PR DESCRIPTION
**Pull request by anayrat**
When several roles exists the window function only report last role.

### CODE REVIEW

**Comment by rjuju** [comment]
Thanks a lot for the PR!  As said in the review, this need a version, do you want to do it or should I?

**Comment by anayrat** [comment]
@rjuju latest changes should fix your review.

File: META.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
    },
    "generated_by": "Julien Rouhaud",
    "meta-spec": {
-      "version": "1.0.0",
+      "version": "1.0.1",
```
Comment (rjuju) [inline META.json]: This is confusing, but it's the version of the pgxn meta spec, not the extension.  The version is automatically updated in the releaze-zip Makefile rule.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Changelog
+=========
+
+XXXXXXXXXX version 1.0.1:
+-------------------------
+
+  **Bug fixes**:
+
+    - Fix missing role settings when several roles exists.
```
Comment (rjuju) [inline CHANGELOG.md]: I guess a slightly more explaining message could be helpful for users.  How about  "Fix issue leading to duplicated role settings changes" or similar?

**Comment by rjuju** [comment]
Thanks, merged!

**Pull request merged**